en/faq.md

```diff
@@ -120,3 +120,11 @@ Three possible reasons:
     ```shell
     kubectl get deployment --all-namespaces |grep tidb-scheduler
     ```
+
+## How does TiDB ensure data safety and reliability?
+
+To ensure persistent storage of data, TiDB clusters deployed by TiDB Operator use [Persistent Volume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) provided by Kubernetes cluster as the storage.
+
+To ensure data safety in case one node is down, PD and TiKV use [Raft Consistency Algorithm](https://raft.github.io/) to replicate the stored data as multiple copies across nodes.
```

Comment:
```suggestion
To ensure data safety in case one node is down, PD and TiKV use [Raft Consistency Algorithm](https://raft.github.io/) to replicate the stored data as multiple replications across nodes.
```